pysal/weights/Distance.py

```diff
@@ -9,11 +9,13 @@
 import scipy.spatial
 from pysal.common import KDTree
 from pysal.weights import W
+from util import get_points_array
 import scipy.stats
 import numpy as np
 
 __all__ = ["knnW", "Kernel", "DistanceBand"]
-
+KDTypes = [scipy.spatial.KDTree, scipy.spatial.cKDTree,
```

Comment:
This was surprising, since `cKDTree` is not a subclass of `KDTree`, so we would have to work a little around this to allow for passing of `cKD` like they're `KD`.